sky/check.py

```diff
@@ -160,15 +220,28 @@ def get_all_clouds():
         # Pretty print for UX.
         if not quiet:
             enabled_clouds_str = '\n  ' + '\n  '.join([
-                _format_enabled_cloud(cloud)
-                for cloud in sorted(all_enabled_clouds)
+                _format_enabled_cloud(cloud) for cloud in sorted(enabled_clouds)
             ])
             echo(f'\n{colorama.Fore.GREEN}{PARTY_POPPER_EMOJI} '
                  f'Enabled clouds {PARTY_POPPER_EMOJI}'
                  f'{colorama.Style.RESET_ALL}{enabled_clouds_str}')
     return enabled_clouds
 
 
+def check(
+    quiet: bool = False,
+    verbose: bool = False,
+    clouds: Optional[Iterable[str]] = None,
+    capability: CloudCapability = CloudCapability.COMPUTE,
```

Comment:
Can we note the rationale for, by default, only checking compute instead of all?